### PR TITLE
GitHub action example

### PR DESCRIPTION
**Pull request by pmiossec**
https://github.com/features/actions

This one:
* is launched at every push (or when tag push if uncommented)
* build & nuget pack (and publish if uncommented)

### CODE REVIEW

File: .github/workflows/dotnetcore.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Build GitExtensions plugin
```
Comment (mast-eu) [inline .github/workflows/dotnetcore.yml]: [nitty]
```suggestion
name: Build Git Extensions plugin
```

**Review by mast-eu — APPROVED** [review]
I like this. :+1:

**Comment by RussKie** [comment]
Nice!
How can we test it?

**Comment by pmiossec** [comment]
>How can we test it?

Merge it ;)

Or you could see the result on my repo of the AzureDevOpsCommitMessage plugin because that's what I used to published the nuget package:
https://github.com/pmiossec/gitextensions.AzureDevOpsCommitMessage/actions

**Comment by mast-eu** [comment]
I had a look here when I approved this PR: https://github.com/pmiossec/gitextensions.plugintemplate/actions?query=workflow%3A"Build+GitExtensions+plugin".
As far as I can see it's basically the same GH Action, right?

**Comment by pmiossec** [comment]
@mast-eu Oh, yes, you're right! I didn't remembered that the action was launched when I push my branch...

**Comment by pmiossec** [comment]
But I have a little more GitHub actions configuration that handle build and release with 2 scripts here:
https://github.com/pmiossec/Gitextensions.AzureDevOpsCommitMessage/tree/master/.github/workflows

Tell me if you think it is interesting and I could add it to the plugin template...

**Comment by RussKie** [comment]
Can we move GE build pipeline to GHA? 😊

**Review by RussKie — COMMENTED** [review]
:shipit:

**Comment by mast-eu** [comment]
@pmiossec : I will definitely come back to you for the GHA configuration 😄

**Pull request merged**